Test/Unit/Helper/ApiTest.php

```diff
@@ -394,13 +404,24 @@ public function sendRequest_withRegularRequest_returnsLocalizedException()
 
         $reportMock = $this->createPartialMock(Report::class, ['setMetaData']);
         $emptyResponseBody = '';
+        // https://github.com/sebastianbergmann/phpunit/issues/3494
+        // Deprecate assertArraySubset()
+        if (class_exists('\PHPUnit\Framework\Constraint\ArraySubset')) {
+            $boltApiRequest = new \PHPUnit\Framework\Constraint\ArraySubset(
+                [
+                    'BOLT API REQUEST' => $request->getData()
+                ]
+            );
+        } else {
+            $boltApiRequest = new \Bolt\Boltpay\Test\Unit\ArraySubset(
+                [
+                    'BOLT API REQUEST' => $request->getData()
+                ]
+            );
+        }
```

Comment:
as this repeats you could extract it in a helper method and put it somewhere generally available

Reply:
Done